garmin_fit.cc

```diff
@@ -25,6 +25,7 @@
 #include <cstdint>             // for uint8_t, uint16_t, uint32_t, int32_t, int8_t, uint64_t
 #include <cstdio>              // for EOF, SEEK_SET, snprintf
 #include <deque>               // for deque, _Deque_iterator, operator!=
+#include <inttypes.h>          // for PRItNN
```

Comment:
this should be <cinttypes> for c++, although the Qt5 -> Qt6 change from int -> qint64 needs to be dealt with